examples/pennylane_run_plugins_hybrid.py

```diff
@@ -116,7 +116,7 @@
 #
 # For this circuit, we will be using the ``'strawberryfields.fock'`` device to construct
 # a QNode. This allows the underlying quantum computation to be performed using the
-# `Strawberry Fields <https://github.com/XanaduAI/strawberryfields>`_ Fock backend.
+# `Strawberry Fields <https://strawberryfields.readthedocs.io>`_ Fock backend.
```

Comment:
Note: the line `dev_fock = qml.device...` below is producing warnings in the built documentation on [readthedocs](https://pennylane.readthedocs.io/en/doc-refactor/tutorials/pennylane_run_plugins_hybrid.html)
```
/home/docs/checkouts/readthedocs.org/user_builds/pennylane/envs/doc-refactor/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:526: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/docs/checkouts/readthedocs.org/user_builds/pennylane/envs/doc-refactor/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:527: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/docs/checkouts/readthedocs.org/user_builds/pennylane/envs/doc-refactor/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:528: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/docs/checkouts/readthedocs.org/user_builds/pennylane/envs/doc-refactor/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:529: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/docs/checkouts/readthedocs.org/user_builds/pennylane/envs/doc-refactor/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:530: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/docs/checkouts/readthedocs.org/user_builds/pennylane/envs/doc-refactor/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:535: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
```

Reply:
Ah, this is a direct result of TF 1.3 using some deprecated behaviour in NumPy 1.17. Perhaps this can be fixed by pinning NumPy 1.16 in the `doc/requirements.txt` file.

Reply:
Added a warnings filter to doc/conf.py +1